js/views/reflowable_view.js

```diff
@@ -176,6 +176,7 @@ ReadiumSDK.Views.ReflowableView = function(options, reader){
 
             //create & append iframe to container frame
             renderIframe();
+            self.trigger(ReadiumSDK.Events.CONTENT_DOCUMENT_UNLOADED, _$iframe, _currentSpineItem );
```

Comment:
Can `_currentSpineItem` be null/undefined here? (in which case the event should not be triggered)